spec/app/domain/authentication/authn_k8s/validate_pod_request_spec.rb

```diff
@@ -87,15 +87,15 @@
     subject(:validator) { Authentication::AuthnK8s::ValidatePodRequest
                             .new(dependencies: dependencies) }
 
-    it 'raises WebserviceNotFound error when webservice is missing' do
+    it 'raises WebserviceNotDefined error when webservice is missing' do
```

Comment:
Why was this renamed from WebserviceNotFound -> WebserviceNotDefined here? Did a search and "WebserviceNotDefined" doesn't exist in other places but this one

Reply:
that's a mistake, it should be `WebserviceNotFound`